gc3apps/gc3.uzh.ch/gridrun.py

```diff
@@ -173,7 +173,7 @@ def parse_args(self):
             if 'from' == kind:
                 gc3libs.utils.test_file(value, os.R_OK,
                                         exception=gc3libs.exceptions.InvalidUsage)
-                self.subst[name] = gc3libs.utils.stripped(open(value, 'r'))
+                self.subst[name] = [i.strip() for i in open(value, 'r').read()]
```

Comment:
I think you want `.readlines()` here -- `.read()` will read the entire contents of a file as a single string.

Reply:
I think actually we may not even need read, as iterating over the open file will get lines. To double check

Reply:
Confirmed that `read().splitlines()` and just plain `open()` both iterate over lines. So I just left the bare open in latest version